enterprise_subsidy/apps/api/v1/views/transaction.py

```diff
@@ -361,21 +362,9 @@ def list(self, request, *args, **kwargs):
         }
         ```
         """
-        base_response = super().list(request, *args, **kwargs)
-
         if self.request.query_params.get("include_aggregates", "").lower() == "true":
-            subsidy = Subsidy.objects.get(uuid=self.requested_subsidy_uuid)
-            aggregates = {
-                "unit": subsidy.unit,
-                "remaining_subsidy_balance": subsidy.current_balance(),
-                "total_quantity": subsidy.ledger.subset_balance(self.get_queryset()),
-            }
-            # Compose a response that combines the base response with additional aggregates data.
-            response_data = {"aggregates": aggregates}
-            response_data.update(base_response.data)
-            return Response(response_data, status=base_response.status_code, headers=base_response.headers)
-        else:
-            return base_response
+            self._paginator = TransactionListAggregatesPaginator()  # pylint: disable=attribute-defined-outside-init
```

Comment:
i suppose one could also make a paginator class which does this routing, calling `TransactionListAggregatesPaginator` when appropriate and something else otherwise. That way one could just define the paginator normally without this call into the (protected) `_paginator`